store/src/java-test/com/zimbra/cs/service/mail/ItemActionTest.java

```diff
@@ -114,16 +115,18 @@ public void copyMessageFromDraftsToSent() throws Exception {
         int msgId = mbox.addMessage(null, pm, dopt, null).getId();
         ItemId iidTarget = new ItemId(mbox, Mailbox.ID_FOLDER_SENT);
         ItemActionHelper op = ItemActionHelper.COPY(null, mbox, SoapProtocol.Soap12, Arrays.asList(msgId), MailItem.Type.MESSAGE, null, iidTarget);
-        Assert.assertNotNull(op);
-        Assert.assertNotNull(op.getResult().getSuccessIds());
-        Assert.assertEquals(1, op.getResult().getSuccessIds().size());
-        ItemId iid = new ItemId(op.getResult().getSuccessIds().get(0), acct.getId());
-        Assert.assertNotNull(iid);
+        Assert.assertNotNull("test non-null response copyMessageFromDraftsToSent", op);
```

Comment:
You don't really need the test name in there - the report has that anyway.  I'm also wary of hard coding test names due to potential future copy/paste issues.  If you need that, JUnit has a mechanism for getting it which is used in many of the tests - see TestJaxb and the use of the TestName class.